parser.cpp

```diff
@@ -1608,6 +1608,10 @@ namespace Sass {
   {
     lex< identifier >();
     string name(lexed);
+    if (name == "compact") {
```

Comment:
Would not incorrectly warn users even if they have defined their own `compact` function? Could we move the warning into the functions.cpp?